DfE.FindInformationAcademiesTrusts/Pages/Shared/_Layout.cshtml

```diff
@@ -15,6 +23,23 @@
   <link rel="apple-touch-icon" href="~/dist/images/govuk-apple-touch-icon.png">
   <link rel="stylesheet" href="~/dist/main.css"/>
   <partial name="_AppInsights"/>
+
+    @if (hasConsented)
+    {
+        <!-- Google Tag Manager -->
+        <script nonce="@nonce">
```

Comment:
Instead of using `@nonce` we would use `asp-add-nonce`. See line 46 for an example.
```suggestion
        <script asp-add-nonce>
```